.flowconfig

```diff
@@ -21,3 +21,4 @@
 esproposal.class_static_fields=enable
 suppress_comment=\\(.\\|\n\\)*\\$FlowIssue
 suppress_comment=\\(.\\|\n\\)*\\$ExpectError
+experimental.const_params=true
```

Comment:
Can we not rely on a global setting? As I understand flow strict mode could be enabled per file.